### PR TITLE
Converted License to MD

### PR DESCRIPTION
**Pull request by NadirJ**

### CODE REVIEW

**Comment by michaelsembwever** [comment]
a lot of text has gone missing … ? 

missing the "Business Source License" initial header.

putting all the parameters into a table does make a big difference for readability, IMHO
e.g. like https://www.hashicorp.com/bsl

File: LICENSE.md [hunk 1]
```diff
@@ -1,30 +1,19 @@
...
+| Change Date: | Two years from the date the Licensed Work is published |
+| Change License: | Apache 2.0 |
+## Updated: 
+24/01/2023
```
Comment (jordanrfrazier) [inline LICENSE.md:14]: What is this dd/mm/yyyy heresy? 

(I would suggest `Updated Jan. 24, 2023`)

**Review by jordanrfrazier — APPROVED** [review]

File: LICENSE.md [hunk 2]
```diff
@@ -1,30 +1,19 @@
...
...ou may make production use of the Licensed Work, provided Your use does not include offering the Licensed Work to third parties on a hosted or embedded basis in order to compete with any of DataStax’s Products.<br>A “competitive offering” is a Product that is offered to third parties, including through paid support arrangements, that significantly overlaps with the capabilities of DataStax’s version of the Licensed Work. If Your Product is not a competitive offering when You first make it generally available, it will not become a competitive offering later due to DataStax releasing a new version of the Licensed Work with additional capabilities.<br>“Product” means software that is offered to end users to manage in their own environments or offered as a service on a hosted basis.<br>“Embedded” means including the source code or executable code from the Licensed Work in a competitive offering. “Embedded” also means packaging the competitive offering in such a way that the Licensed Work must be accessed or downloaded for the competitive offering to operate.<br>Hosting or using the Licensed Work(s) for internal purposes within an organization is not considered a competitive offering. |
```
[the remark this answers is not in the document]
Comment (NadirJ) [reply LICENSE.md:10]: new lines break the table

**Review by NadirJ — COMMENTED** [review]
> either line 6 or 7 is adding an extra empty row 

this is a limitation of Markdown I guess, I want a table with no headings

**Review by michaelsembwever — APPROVED** [review]

**Comment by michaelsembwever** [comment]
thanks @NadirJ , the license is far more intuitive at a glance now 👍

**Pull request merged**